spec/requests/authorized_family_members_requests_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+RSpec.describe "Authorized Family Members Controller", type: :request do
+  let(:partner) { create(:partner) }
+  let(:user) { create(:user, partner: partner) }
+  let!(:family) { create(:family, partner: partner) }
+
+  before do
+    sign_in(user)
+  end
+
+  describe "GET #new" do
+    it "returns http status ok" do
+      get new_authorized_family_member_path
```

Comment:
```suggestion
      get new_authorized_family_member_path(family_id: family.id)
```
The current way the endpoint knows which family to add an authorized family member is through a query parameter like this:
```
http://partner.test/authorized_family_members/new?family_id=7
```

Thus it is raising a 500 error without it because `family` will be undefined or nil in the code:
```
  def new
    @authorized_family_member = family.authorized_family_members.new
  end
```